### DISCUSSION TITLE
Does Samsung tool also officially support?

### DISCUSSION THREAD

**Discussion by DusJson** [openthread/openthread#9020] [Q&A] [answered]
Hi

According to readme, Samsung is also on the support list.
I wonder if the tool made bySamsung is also supportedby openthread.
If so, why this tool exists in Samsung repo apart from openthread repo.

The tool repo is
github.com/Samsung/ioter

**Comment by sangsoolee** [comment, marked as answer]
Hi @DusJson 
The emulator application was created by me. I don't think it would even make sense to be part of OpenThread since it's more of a Matter tool.

The application complies with the Thread spec and uses the OpenThread code.
If there is a problem with use, improvement, or other opinion, it would be nice to ask a question to the repo of the tool.
https://github.com/Samsung/ioter/discussions
